docs/releases/4.0.txt

```diff
@@ -288,6 +288,11 @@ Models
 * :class:`~django.db.models.Lookup` expressions may now be used in ``QuerySet``
   annotations, aggregations, and directly in filters.
 
+* Aggregate functions now support a ``default`` argument that allows specifying
+  a value to be returned when the queryset (or grouping) contains no entries,
+  rather than ``None``. It does this by wrapping the aggregate and the provided
+  expression with :class:`~django.db.models.functions.Coalesce`.
```

Comment:
The last sentence is not necessary in release notes, I'd also reword it a bit
```suggestion
* The new :ref:`default <aggregate-default>` argument for built-in aggregates
  allows specifying a value to be returned when the queryset (or grouping)
  contains no entries, rather than ``None``.
```